sema/discovery/discovery.py

```
@@ -152,6 +166,35 @@
             )
         return False
 
+    def _parse_url_content(self, resp_text: str, resp_url: str):
```

Ruff

Missing return type annotation for private function `_parse_url_content`

(ANN202)